imagetest/test_suites/security/image_security_test.go

```diff
@@ -128,6 +128,11 @@ func TestPasswordSecurity(t *testing.T) {
 		// SLES ships with "PermitRootLogin yes" in SSHD config.
 		t.Skip("Not supported on SLES")
 	}
+	if strings.Contains(image, "ubuntu") {
+		// Ubuntu doesn't set this option, but Ubuntu sshd defaults to prohibit-password
+		// TODO: find a way to determine the default so we can support 'not set' generally.
```

Comment:
I'm not sure about our open-source code on GitHub, but usually we try to leave a username or bug ticket number on TODOs so they're easier to track.

Reply:
+1 to @ericdand's comment